ansible/playbooks/roles/node/templates/harmony.service.j2

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=harmony service
+After=network.target
+StartLimitIntervalSec=0
```

Comment:
this is for systemd > 219 version, on AWS the systemd version is older, and can't support this variable, it is called `StartLimitInterval`

Reply:
got it. i will make it compatible